DER/DerivativesContracts/Options.rdf

```diff
@@ -370,6 +560,77 @@
 		<rdfs:label xml:lang="en">floating lookback strike terms</rdfs:label>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-der-opt;ForeignCurrencyOption">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;ExchangeTradedOption"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;ForeignExchangeDerivative"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;FxOptionUnderlying"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-dbt-ex;hasExerciseTerms"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;CurrencyOptionExerciseTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">foreign currency option</rdfs:label>
+		<skos:definition xml:lang="en">An option on a currency, based on the spot exchange rate of that currency to the currency of the security itself.</skos:definition>
+		<skos:editorialNote xml:lang="en">Definition needed - this is just a place holder. Actually, this would also have a Currency Future as the Underlying Action: change this. Definition:Review</skos:editorialNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;FutureOptionsExerciseTerms">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ex;AmericanExerciseTerms"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-der-opt;delivery"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-fi;Future"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">future options exercise terms</rdfs:label>
+		<skos:definition xml:lang="en">Contractual terms defining when or under what circumstances an options instrument may be exercised.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Also includes terms defining delivery when the contract is exercised.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;FxOptionUnderlying">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-bsc;Underlier"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+				<owl:someValuesFrom rdf:resource="&fibo-ind-fx-fx;CurrencySpotBuyRate"/>
```

Comment:
No it's not going to be a specific rate - this is for "an exchange rate quoted at a specific point in time"
Also why the Buy rate?